src/main/java/jeff/exception/JeffException.java

```diff
@@ -0,0 +1,36 @@
+package jeff.exception;
+
+/**
+ * Represents an exception for any errors in this package.
+ */
+public class JeffException extends Exception {
+    private final String message;
+
+    /**
+     * Constructor that creates a JeffException object with a specific message.
+     *
+     * @param message that describes the error
+     */
+    public JeffException(String message) {
+        super(message);
+        this.message = message;
+    }
+
+    /**
+     * Constructor that creates a JeffException object with a general message.
+     */
+    public JeffException() {
+        super();
+        this.message = "Sorry, I don't know what this means.";
+    }
+
+    /**
+     * Returns the message of the error.
+     *
+     * @return the error message
+     */
+    @Override
+    public String toString() {
+        return this.message;
+    }
+}
```

Comment:
I like how you created a custom exception for the chatbot—it'll help with managing current (and future) exceptions in a more precise manner!